indra/assemblers/indranet/assembler.py

```diff
@@ -281,9 +369,173 @@ def make_df(self, exclude_stmts=None, complex_members=3,
+            ...     return G
+
+        Returns
+        -------
+        model : IndraNet
+            IndraNet graph object.
+        """
+        # Filter out statements with one agent or with None subject
+        stmts = [stmt for stmt in self.statements if len(
+            [ag for ag in stmt.agent_list() if ag is not None]) > 1]
+        if exclude_stmts:
+            exclude_types = tuple(
+                get_statement_by_name(st_type) for st_type in exclude_stmts)
+            stmts = [stmt for stmt in stmts
+                     if not isinstance(stmt, exclude_types)]
+        stmts = _store_edge_data(stmts, extra_columns)
+        if graph_type == 'signed':
+            if not sign_dict:
+                sign_dict = default_sign_dict
+            graph_stmts = []
+            for stmt_type in sign_dict:
+                graph_stmts += ac.filter_by_type(stmts, stmt_type)
+            # Conversion statements can also be turned into two types of signed
+            conv_stmts = ac.filter_by_type(stmts, Conversion)
+            for stmt in conv_stmts:
+                for obj in stmt.obj_from:
+                    graph_stmts.append(
+                        DecreaseAmount(stmt.subj, obj, stmt.evidence))
+                for obj in stmt.obj_to:
+                    graph_stmts.append(
+                        IncreaseAmount(stmt.subj, obj, stmt.evidence))
+            graph_stmts = ac.run_preassembly(
+                graph_stmts, return_toplevel=False,
+                belief_scorer=belief_scorer,
+                matches_fun=partial(agent_name_polarity_matches,
+                                    sign_dict=sign_dict))
+            G = nx.MultiDiGraph()
+        elif graph_type in ['unsigned', 'multi_graph']:
+            complex_stmts = ac.filter_by_type(stmts, Complex)
+            conv_stmts = ac.filter_by_type(stmts, Conversion)
+            graph_stmts = [stmt for stmt in stmts if stmt not in complex_stmts
+                           and stmt not in conv_stmts]
+            for stmt in complex_stmts:
+                agents = [ag for ag in stmt.agent_list() if ag is not None]
+                if len(agents) > complex_members:
+                    continue
+                for a, b in permutations(agents, 2):
+                    graph_stmts.append(IncreaseAmount(a, b, stmt.evidence))
+            for stmt in conv_stmts:
+                for obj in stmt.obj_from:
+                    graph_stmts.append(
+                        DecreaseAmount(stmt.subj, obj, stmt.evidence))
+                for obj in stmt.obj_to:
+                    graph_stmts.append(
+                        IncreaseAmount(stmt.subj, obj, stmt.evidence))
+            if graph_type == 'unsigned':
+                graph_stmts = ac.run_preassembly(
+                    graph_stmts, return_toplevel=False,
+                    belief_scorer=belief_scorer,
+                    matches_fun=agent_name_stmt_matches)
```

Comment:
I wonder if we should also take refinements into account here, without also supplying a custom `refinement_fun` along with a custom `matches_fun`, issues could come up. Alternatively, if we want to cleanly ignore refinements here, we should pass the `run_refinement=False` argument here.